libcomposefs/lcfs-writer-erofs.c

```diff
@@ -36,6 +36,121 @@
 #include <assert.h>
 #include <linux/fsverity.h>
 
+/* The xxh32 hash function is copied from xxhash with this copyright:
```

Comment:
From *where* precisely?  A git URL and exact digest would be useful.

Taking the copy from the Linux kernel and massaging it to compile may not actually be *too* hard...

Reply:
I took it from:
https://www.spinics.net/lists/kernel/msg4859620.html
Which is taken from the kernel, but initially was from 
static const uint32_t PRIME32_1 = 2654435761U;

Reply:
https://github.com/torvalds/linux/blob/master/lib/xxhash.c

Reply:
OK makes sense, but I more meant let's add this data into the comment in the code, so someone knows where to look for updates.